skills/skill-pressure-test/SKILL.md

```
@@ -0,0 +1,107 @@
+---
+name: skill-pressure-test
+description: "Interrogate a plan, decision, or design one question at a time until it holds — use to stress-test your own thinking before committing to it"
+---
+
+> **Host: Codex CLI** — This skill was designed for Claude Code and adapted for Codex.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)